---
float32 dist
float32 avg_vel
